sematic/resolvers/silent_resolver.py

```diff
@@ -19,7 +20,14 @@ def _run_inline(self, future: AbstractFuture) -> None:
         self._set_future_state(future, FutureState.SCHEDULED)
         try:
             self._start_inline_execution(future.id)
-            value = future.calculator.calculate(**future.resolved_kwargs)
+            with set_context(
+                SematicContext(
+                    id=future.id,
+                    root_id=self._root_future.id,
+                    resolver_class_path=self.classpath(),
+                )
+            ):
+                value = future.calculator.calculate(**future.resolved_kwargs)
```

Comment:
Does this and the change in `worker.py` cover all cases? If yes, then our code is 👌 😎

Reply:
it does indeed!